# Pathogen Genomic Sequencing Resources
Lectures, papers, and online resources for pathogen sequencing.

## Lectures

**The NGS Toolbox for Arbovirologists: Do I Need a Screwdriver or a Hammer?**
- By Nathan Grubaugh, presented during the 2019 ASTMH Pre-meeting workshop "Arboviromics: How ‘omics’ Technologies are Advancing Arbovirus Research and Control" in Washington, DC.
- Describes what to consider when selecting sequencing approaches and platforms.
- File = (2019.11.20_ACAV-premeeting_Grubaugh.pptx)

**Grubaugh Lab NGS Sample Prep Protocols**
- By Joseph Fauver, presented during 2019.05.03 Grubaugh Lab meeting in New Haven, CT
- Describes details of what happens at each step of library prep and sequencing.
- File = [2019.05.03_NGS_protocols_Fauver.pptx](https://github.com/grubaughlab/path-gen_resources/blob/master/2019.05.03_NGS-protocols_Fauver.pptx)

**Principles of Molecular Epidemiology**
- By Nathan Grubaugh, presented during the 2019 EpiMol workshop in Salvador, Brazil
- Describes the fundamentals of pathogen genomic epidemiology, phylogenetic methods, and probability errors and validity.
- File = [2019.07.23_principles-in-mol-epi_Grubaugh.pptx](https://github.com/grubaughlab/path-gen_resources/blob/master/2019.07.23_principles-in-mol-epi_Grubaugh.pptx)

**Practices in Molecular Epidemiology**
- By Nathan Grubaugh, presented during the 2019 EpiMol workshop in Salvador, Brazil
- Describes what you can learn by reading a phylogenetic tree.
- File = [2019.07.23_practices-in-mol-epi_Grubaugh.pptx](https://github.com/grubaughlab/path-gen_resources/blob/master/2019.07.23_practices-in-mol-epi_Grubaugh.pptx)

## Online resources

**[Grubaugh Lab: Amplicon-based virus sequencing using Illumina and MioION platforms](http://grubaughlab.com/open-science/amplicon-sequencing/)**
- Detailed protocols for designing amplicon schemes, preparing the libraries, and analyzing the data.

**[Grubaugh Lab: Protocols in development](http://grubaughlab.com/open-science/protocols/)**
- Links to metagenomic, amplicon, and other sequencing protocols in development.

**[Artic Network: Virus specific resources for MinION sequencing](https://artic.network/resources)**
- Detailed list of reagents/equipment, sequencing protocols, and workflows setting up laptops, bioinformatic analysis, & data viz. 

**[The ZiBRA Pipeline for Zika virus sequencing using the MinION](http://www.zibraproject.org/protocol/)**
- Detailed protocols for sequencing, basecalling, and analysis.

## Technical papers

**[An amplicon-based sequencing framework for accurately measuring intrahost virus diversity using PrimalSeq and iVar](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-018-1618-7)**
- Grubaugh et al., 2019. *Genome Biology*

**[Multiplex PCR method for MinION and Illumina sequencing of Zika and other virus genomes directly from clinical samples](https://www.nature.com/articles/nprot.2017.066)**
- Quick et al., 2017. *Nature Protocols*

**[Enhanced methods for unbiased deep sequencing of Lassa and Ebola RNA viruses from clinical and biological samples](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-014-0519-7)**
- Matranga et al., 2014. *Genome Biology*

**[Library preparation for highly accurate population sequencing of RNA viruses](https://www.nature.com/articles/nprot.2014.118)**
- Avevedo & Andino, 2014. *Nature Protocols*

**[Capturing sequence diversity in metagenomes with comprehensive and scalable probe design](https://www.nature.com/articles/s41587-018-0006-x)**
- Metsky et al., 2019. *Nature Biotechnology*


